wallet/res/xml/preference_settings.xml

```diff
@@ -14,6 +14,11 @@
         android:summary="@string/preferences_own_name_summary"
         android:title="@string/preferences_own_name_title" />
 
+	<CheckBoxPreference
+		android:defaultValue="false"
+		android:key="hide_balance"
+		android:title="@string/preferences_hide_balance" />
+
```

Comment:
This is obsolete, but the current designs don't say where this setting should be in the UI.

Reply:
@HashEngineering is the new design PR merged already?

Reply:
Yes, it is.  The Menu PR has been merged.

you can look at this to get an idea of how to add menu items:  #258